src/cascadia/TerminalSettingsModel/userDefaults.json

```diff
@@ -1,75 +1,32 @@
-// This file was initially generated by %PRODUCT% %VERSION%
-// It should still be usable in newer versions, but newer versions might have additional
-// settings, help text, or changes that you will not see unless you clear this file
-// and let us generate a new one for you.

```

Comment:
Since I use `CascadiaSettings::ToJson` for the initial creation of settings.json, comments aren't supported anymore. That's why I removed them from `userDefaults.json`.

Reply:
Rioght -- since most people are going to be using settings through the UI nowadays I don't feel the need to include heavy documentation comments in the file itself. It was our ONLY user interface before, and now it is simply an auxiliary one.